helpers/minio_helpers.py

```diff
@@ -199,5 +201,33 @@ def get_files_and_last_modified(self, prefix: str):
         logging.info(f"*****List of files: {file_info_list}")
         return file_info_list
 
+    def read_json_file(self, dest_path, dest_name):
```

Comment:
If possible to use existing methods within the class and add helpers for temp file creation. that would be great !